.github/workflows/changelog_check.yml

```
@@ -0,0 +1,13 @@
+Name: Changelog-Checks
+on: [pull_request]
+
+jobs:
+  check-newsfile:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
```

```suggestion
          python-version: "3.7"
```

And in the other pipeline, so that we're consistent with our strings.